Course/Course/Presentation/Dates/CourseDatesView.swift

```diff
@@ -46,10 +49,33 @@ public struct CourseDatesView: View {
                         viewModel: viewModel,
                         coordinate: $coordinate,
                         collapsed: $collapsed,
+                        viewHeight: $viewHeight,
                         courseDates: courseDates,
                         courseID: courseID
                     )
                     .padding(.top, 10)
+                } else {
+                    GeometryReader { proxy in
+                        VStack {
+                            ScrollView {
+                                DynamicOffsetView(
+                                    coordinate: $coordinate,
+                                    collapsed: $collapsed,
+                                    viewHeight: $viewHeight
+                                )
+                                
+                                FullScreenErrorView(
+                                    type: .noCourseDates(
+                                        error: CourseLocalization.Error.courseDateUnavailable,
+                                        image: CoreAssets.information.swiftUIImage
+                                    )
+                                )
```

Comment:
Is it possible to configure FullScreenErrorView in a way that we can directly use it like we are using OfflineSnackBarView?